app/assets/javascripts/additional_details/item_options.coffee

```diff
@@ -33,7 +33,11 @@ $ ->
     if showOptions($(value).val(), needOptions)
       $(".item-options[data-item-form-id=#{itemFormId}]").removeClass('hidden')
 
+
   $('#datetimepicker').datetimepicker()
 
   $('.selectpicker').selectpicker()
 
+$('.questionnaires.create').ready ->
+  $.each $('.option-content:visible'), (key, value) ->
+    $(value).parents('.option-input').find('input:hidden').val('true')
```

Comment:
Can you make the query for the hidden field more specific? This could bite later if a sibling hidden field is added.